vsutil/func.py

```diff
@@ -142,3 +147,14 @@ def iterate(base: T, function: Callable[[Union[T, R]], R], count: int) -> Union[
     for _ in range(count):
         v = function(v)
     return v
+
+
+def normalize_franges(franges: FrameRange, /) -> Iterable[int]:
+    if isinstance(franges, int):
+        return [franges]
+    elif isinstance(franges, tuple):
+        start, stop = franges
+        step = -1 if stop < start else 1
+        return range(start, stop + step, step)
+    else:
+        return franges
```

Comment:
I said before that we’re not doing this, especially not as part of the public API.

Reply:
Can you point me to your exact reasoning? Standardizing a way of frame specification sounds like a fine usage of `vsutil`, provided it also has a way of making the frame specification usable e.g. by returning an iterable over all frame numbers.